### PR TITLE
[New Rule] Potential Masquerading as Communication Apps

### PR DESCRIPTION
**Pull request by w0rk3r**
## Summary

Identifies suspicious instances of communications apps, both unsigned and renamed ones, that can indicate an attempt to conceal malicious activity, bypass security features such as allowlists, or trick users into executing malware.

### CODE REVIEW

**Review by imays11 — APPROVED** [review]

**Review by Samirbous — APPROVED** [review]

File: rules/windows/defense_evasion_masquerading_communication_apps.toml [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+"""
+from = "now-9m"
+index = ["logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Potential Masquerading as Communication Apps"
+risk_score = 47
+rule_id = "c9482bfa-a553-4226-8ea2-4959bd4f7923"
+severity = "medium"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Defense Evasion"]
```
Comment (w0rk3r) [inline rules/windows/defense_evasion_masquerading_communication_apps.toml:23]:
```suggestion
tags = ["Domain: Endpoint", "OS: Windows", "Use Case: Threat Detection", "Tactic: Defense Evasion"]
```

File: rules/windows/defense_evasion_masquerading_communication_apps.toml [hunk 2]
```diff
@@ -0,0 +1,87 @@
+[metadata]
+creation_date = "2023/05/05"
+integration = ["endpoint"]
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2023/05/05"
```
Comment (w0rk3r) [inline rules/windows/defense_evasion_masquerading_communication_apps.toml:7]:
```suggestion
updated_date = "2023/06/30"
```

File: rules/windows/defense_evasion_masquerading_communication_apps.toml [hunk 3]
```diff
@@ -0,0 +1,87 @@
+[metadata]
+creation_date = "2023/05/05"
+integration = ["endpoint"]
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2023/06/30"
```
Comment (w0rk3r) [inline rules/windows/defense_evasion_masquerading_communication_apps.toml:7]:
```suggestion
updated_date = "2023/06/30"
bypass_bbr_timing = true
```

File: rules/windows/defense_evasion_masquerading_communication_apps.toml [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+conceal malicious activity, bypass security features such as allowlists, or trick users into executing malware.
+"""
+from = "now-9m"
+index = ["logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Potential Masquerading as Communication Apps"
+risk_score = 47
+rule_id = "c9482bfa-a553-4226-8ea2-4959bd4f7923"
+severity = "medium"
```
Comment (w0rk3r) [inline rules/windows/defense_evasion_masquerading_communication_apps.toml:22]:
```suggestion
risk_score = 21
rule_id = "c9482bfa-a553-4226-8ea2-4959bd4f7923"
severity = "low"
```
Will bump the severity once I move it from BBR to prod

**Pull request merged**